src/main/java/tatracker/logic/parser/FindCommandParser.java

```diff
@@ -21,8 +21,7 @@
     public FindCommand parse(String args) throws ParseException {
         String trimmedArgs = args.trim();
         if (trimmedArgs.isEmpty()) {
```

Comment:
if u plan to delete findcommand.java, remeber to delete this too

Reply:
Okay!